Bottom: 3b80147c1b2077d6f2aae77504ca48a262713794
Top:    c01c449ce744ef4ed5a591d6212c8668678a6dda
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 23:48:13 +0100

hid/gcode: Fixup warnings due to assigned but unused variables


---

diff --git a/src/hid/gcode/trace.c b/src/hid/gcode/trace.c
index 8d7e5e6..7fb9012 100644
--- a/src/hid/gcode/trace.c
+++ b/src/hid/gcode/trace.c
@@ -1406,10 +1406,7 @@ plotpolygon (privpath_t * pp, FILE * f, double scale)
   int i;
   int m = pp->m;
   int *po = pp->po;
-  int n = pp->len;
   point_t *pt = pp->pt;
-  int x0 = pp->x0;
-  int y0 = pp->y0;
   /* double scale=1.0/dpi; */
   double dm = 0;
 
@@ -1417,10 +1414,7 @@ plotpolygon (privpath_t * pp, FILE * f, double scale)
     return 0;
 
   po = pp->po;
-  n = pp->len;
   pt = pp->pt;
-  x0 = pp->x0;
-  y0 = pp->y0;
 
   fprintf (f, "G0 X%f Y%f    (start point)\n", pt[po[0]].x * scale,
 	   pt[po[0]].y * scale);
